### ISSUE TITLE
Expose version/hash/build date

### ISSUE THREAD

**Issue by techtonik** [alex2108/syncthing-tray#26]
I can't determine which version of `syncthing-tray` I am running. It doesn't contain any identification info.

**Comment by alex2108** [comment]
yeah that is annoying sometimes, will add something (probably during the next 2 weeks)

**Comment by techtonik** [comment]
`syncthing` uses shell build script that fetches version `RELEASE` file if present or uses git hash. That probably not going to work on Windows.

**Comment by alex2108** [comment]
Fixed in newest release, date and version are manually specified when compiling (see readme.md)
